content/en/blog/_posts/2023-09-15-userns-stateful-pods/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: blog
+title: "User namespaces: Now alpha support to run stateful pods too!"
+date: 2023-09-15
```

Comment:
Please use
```suggestion
date: 2023-09-13
```
and rename the file accordingly.

Reply:
Perfect, thanks!